x-pack/solutions/observability/packages/kbn-observability-schema/related_dashboards/schema/recommended_dashboard/v1.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from '@kbn/zod';
+import { relevantPanelSchema } from '../relevant_panel/latest';
+
+export const relatedDashboardSchema = z.object({
+  id: z.string(),
+  title: z.string(),
+  matchedBy: z.object({
```

Comment:
If the same schema is use by linked dashboards as well, shouldn't `matchedBy` be optional? Or are all these properties needed by the linked dashboards as well? Can you explain?

Reply:
Hmm... I don't know really. Probably. You could imagine maybe we used matchedBy: { linked: true }, or something like that for linked.

Ultimately, feel free to change this schema however you feel is appropriate once you start working on linked dashboards.